lib/core/src/Cardano/Wallet/DB/Sqlite/TH.hs

```diff
@@ -176,6 +176,7 @@ ProtocolParameters
     protocolParametersDecentralizationLevel Percentage      sql=decentralization_level
     protocolParametersDesiredNumberOfPools  Word16          sql=desired_pool_number
     protocolParametersMinimumUtxoValue      W.Coin          sql=minimum_utxo_value
+    protocolParametersKeyDeposit            W.Coin          sql=key_deposit
```

Comment:
Sadly, this needs to be added as the very last. `persistent` does not "tolerate" adding new columns in the middle of existing declarations.